lib/active_admin/resource_controller/streaming.rb

```diff
@@ -24,14 +24,14 @@ def stream_resource(&block)
         headers["Last-Modified"] = Time.current.httpdate
 
         if ActiveAdmin.application.disable_streaming_in.include? Rails.env
-          self.response_body = block[String.new]
+          self.response_body = block[String.new] # rubocop:disable Performance/UnfreezeString to preserve encoding
         else
           self.response_body = Enumerator.new &block
         end
       end
 
       def csv_filename
-        "#{resource_collection_name.to_s.gsub('_', '-')}-#{Time.zone.now.to_date.to_formatted_s(:default)}.csv"
+        "#{resource_collection_name.to_s.tr('_', '-')}-#{Time.zone.now.to_date.to_formatted_s(:default)}.csv"
```

Comment:
The merge conflict is on this line since `to_formatted_s(:default)` was changed to `to_s` in #8119.